.github/workflows/ci.yml

```diff
@@ -205,9 +210,15 @@ jobs:
           output-folder: dist-artifacts
           access-policy-token: ${{ env.SIGN_PLUGIN_ACCESS_POLICY_TOKEN }}
 
+      - name: Trufflehog secrets scanning
+        if: ${{ inputs.run-trufflehog == true }}
+        uses: grafana/plugin-ci-workflows/actions/plugins/trufflehog@main
+        with:
+          folder: dist-artifacts
```

Comment:
This will scan all the packaged zip files before uploading them. Do we also want to run trufflehog on the source code itself?

Reply:
no, one the packaged zip file should be enough at first.